src/renderer/src/components/editor/FileEditor/hooks/useTiptapEditor.ts

```
@@ -0,0 +1,62 @@
+import { useEditor } from '@tiptap/react'
+import StarterKit from '@tiptap/starter-kit'
+import Underline from '@tiptap/extension-underline'
+import Placeholder from '@tiptap/extension-placeholder'
+import CharacterCount from '@tiptap/extension-character-count'
+import { useEffect, useState } from 'react'
+import { FormatState } from '@renderer/types/editor/clinet'
```

ESLint

[error] `@renderer/types/editor/clinet` import should occur before import of `@tiptap/react`

(import/order)